dump_webcompat_to_db.py

```diff
@@ -47,7 +60,7 @@ def main():
     # stuff data into database..
     for bug in data['bugs']:
         db_session.add(
-            Issue(bug['id'], bug['summary'], bug['url'], bug['body']))
+            Issue(bug['id'], bug['summary'], bug['url'], extract_domain_name(bug['url']), bug['body'], bug['state'], bug['creation_time'], bug['last_change_time']))

```

Comment:
Can you run pep8 on this file? This doesn't follow webcompat style guidelines.

https://github.com/webcompat/webcompat.com/blob/master/CONTRIBUTING.md#python